builder/dockerfile/dispatchers.go

```diff
@@ -187,6 +190,7 @@ func dispatchCopy(req dispatchRequest) error {
 	if err != nil {
 		return err
 	}
+	copyInstruction.flChown = flChown.Value
```

Comment:
I think we should parse the Value here so we can fail fast when it's wrong. That way we can store an `IDPair` in `copyInstruction` as well.